src/dataServices/dynamoDbDataService.test.ts

```diff
@@ -948,3 +948,47 @@ each(['cancelExport', 'getExportStatus']).test('%s:Unable to find job', async (t
         );
     }
 });
+
+describe('getActiveSubscriptions', async () => {
```

Comment:
```suggestion
test('getActiveSubscriptions', async () => {
```

Make sure to check Jest output when writing new tests:
```
Returning a Promise from "describe" is not supported. Tests must be defined synchronously.
Returning a value from "describe" will fail the test in a future version of Jest.
```

Reply:
Thanks for catching that